config/manager/kustomization.yaml

```diff
@@ -12,5 +12,5 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 images:
 - name: controller
-  newName: quay.io/vhire/opendatahub-operator
-  newTag: dev-0.0.1
+  newName: registry.jharmison.com/library/opendatahub-operator
```

Comment:
This is probably left over from testing but I assume this shouldn't be merged.

Reply:
Yes, will revert this change

Reply:
Oops - apologies :)